bandc_c+a.rb

```diff
@@ -1,27 +1,43 @@
 puts "Welcome! Select classic or advanced game! For classic press 'c', for advanced press 'a'."
 game_type = gets.chomp
 
-while game_type == "c" or "a"
+def classic_check(variable)
+  if variable.size != 4
+    puts "Please, try again. Your guess must contain exactly 4 digits!"
+  end
+        
+  if variable.uniq.length != 4
+    puts "Please, try again. Your guess must contain 4 unique digits!"
+  end
+    
+  if variable.include?(0)
+    puts "Please, try again. You cannot use zero in your guess!"
+  end
```

Comment:
Well, if you are deciding to skip or not to skip an iteration, then you need some boolean reslut, don't you?